### PR TITLE
Change list view header action menu from select to dropdown

### LINKED ISSUE #171: Fix bleed issue with "Download Adapter" item
See PR #164. "Download Adapter" is cut off.

### PR DESCRIPTION
**Pull request by srambach**
Fixes #171 
The action menu should have been a dropdown rather than select, and a dropdown allows a right alignment, so this fixes the overflow off the right side as well as correcting the semantics of the element.

To verify, look at the ViewHeader story and also the Client settings page (actions menu in the header)

### CODE REVIEW

File: src/components/view-header/ViewHeader.tsx [hunk 1]
```diff
@@ -81,19 +87,17 @@ export const ViewHeader = ({
...
-                      onSelect={(_, value) => {
-                        if (onSelect) {
-                          onSelect(value as string);
-                        }
-                        setOpen(false);
-                      }}
-                    >
-                      {selectItems}
-                    </Select>
+                    {/* This is the dropdown to replace the select.  It operates but does not initiate the actions. See it by going to Clients and click on a client. All 3 menu actions should work but don't. */}
```
Comment (jenny-s51) [inline src/components/view-header/ViewHeader.tsx:90]: delete this line

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Looking good @srambach ! Actions menu working as expected per @mfrances17 . Just delete that comment above :)

**Review by edewit — APPROVED** [review]
:tada: so much better

**Review by jenny-s51 — APPROVED** [review]
👍

**Pull request merged**